source/org/zfin/marker/repository/HibernateMarkerRepository.java

```diff
@@ -814,6 +815,50 @@ public void addMarkerRelationshipAttribution(MarkerRelationship mrel, Publicatio
         infrastructureRepository.insertUpdatesTable(mrel.getSecondMarker(), "", "new attribution, marker relationship: " + mrel.getZdbID() + " with pub: " + attributionZdbID, attributionZdbID, "");
     }
 
+
+    public GenomeLocation addGenomeLocation(GenomeLocation genomeLocation) {

```

Comment:
I am not sure if you can persist new / updated GenomeLocation objects. That class maps to 
sequence_feature_chromosome_location_generated
and records in there may get overwritten. We need to check the DB functions that are responsible for getting data from sequence_feature_chromosome_location to sequence_feature_chromosome_location_generated

Reply:
I ran the jenkins job (regenerate-chromosome-mart) on the cell vm and found that it did not overwrite the data I am storing in that warehouse table. This makes sense when digging into that job because the delete statements all seem to be conditional on the source column, like shown below.  However, I'm not sure if that job is even run anymore?

```
delete from sequence_feature_chromosome_location_generated
 where sfclg_location_source = 'ZfinGbrowseStartEndLoader';
```